src/components/PeerGroupsUpdate.tsx

```diff
@@ -0,0 +1,176 @@
+import React, {useEffect, useRef, useState} from 'react';
+import {useDispatch, useSelector} from "react-redux";
+import {RootState} from "typesafe-actions";
+import { actions as peerActions } from '../store/peer';
+import {
+    Col,
+    Row,
+    Typography,
+    Space,
+    Button, Drawer, Form, Select, Tag
+} from "antd";
+import type { CustomTagProps } from 'rc-select/lib/BaseSelect'
+import {useAuth0} from "@auth0/auth0-react";
+import {PeerGroupsToSave} from "../store/peer/types";
+import {Group, GroupPeer} from "../store/group/types";
+
+const { Paragraph } = Typography;
+const { Option } = Select;
+
+const PeerGroupsUpdate = () => {
+    const { getAccessTokenSilently } = useAuth0()
+    const dispatch = useDispatch()
+    const groups =  useSelector((state: RootState) => state.group.data)
+    const peer =   useSelector((state: RootState) => state.peer.peer)
+    const updateGroupsVisible = useSelector((state: RootState) => state.peer.updateGroupsVisible)
+    const savedGroups = useSelector((state: RootState) => state.peer.savedGroups)
+
+    const [tagGroups, setTagGroups] = useState([] as string[])
+    const [selectedTagGroups, setSelectedTagGroups] = useState([] as string[])
```

Comment:
Codacy has a fix for the issue: [Replace `····const·[selectedTagGroups,·setSelectedTagGroups]·=·useState([]·as·string[])` with `··const·[selectedTagGroups,·setSelectedTagGroups]·=·useState([]·as·string[]);`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
  const [selectedTagGroups, setSelectedTagGroups] = useState([] as string[]);
```